src/databricks/labs/ucx/framework/utils.py

```diff
@@ -29,8 +29,9 @@ def escape_sql_identifier(path: str, optional: bool | None = True) -> str:
     return ".".join(escaped)
 
 
-def run_command(command: str) -> tuple[int, str, str]:
-    logger.info(f"Invoking command: {command}")
-    with subprocess.Popen(command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE) as process:
+def run_command(command: str | list[str]) -> tuple[int, str, str]:
+    args = command.split() if isinstance(command, str) else command
+    logger.info(f"Invoking command: {' '.join(args)}")
```

Comment:
I think in terms of understanding _exactly_ what is being executed I'd prefer:
```suggestion
    logger.info(f"Invoking command: {args!r}")
```